src/devices/bus/pce/pce_slot.h

```diff
@@ -96,11 +99,9 @@ class pce_cart_slot_device : public device_t,
 
 	void set_intf(const char * interface) { m_interface = interface; }
 
-	// reading and writing
-	uint8_t read_cart(offs_t offset);
-	void write_cart(offs_t offset, uint8_t data);
-
 protected:
+	required_address_space m_address_space;
```

Comment:
Please put this with the other data members (below the member function declarations).